Backend/app/dashboard/Lib/src/Domain/Types/Merchant.hs

```diff
@@ -11,21 +12,78 @@
 
  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
 -}
+{-# LANGUAGE InstanceSigs #-}
```

Comment:
Above pragmas look unused

Reply:
No , below there is a  explicit type signatures in instance declarations.

```